xonsh/execer.py

```diff
@@ -8,7 +8,7 @@
 import warnings
 from collections import Mapping
 
-from xonsh import ast
+from xonsh.ast import CtxAwareTransformer
```

Comment:
So *most* of the modifications were to the second case.  We already happened to be coding in the style for the same package imports :)